ChangeLog.d/issue4564.txt

```diff
@@ -0,0 +1,3 @@
+Removals
```

Comment:
These changes are purely internal and not visible to normal users, so they shouldn't have a ChangeLog entry.